test/integration/orchestrator-spec.ts

```diff
@@ -818,10 +818,7 @@ describe("Orchestrator", () => {
                         actions: [
                             [new CallActivityWithRetryAction("Hello", retryOptions, "World")],
                         ],
-                        output: [
-                            startingTime,
-                            moment(startingTime).add(1, "m").add(30, "s").toDate(),
-                        ],
+                        output: [startingTime, moment(startingTime).add(1, "m").toDate()],

```

Comment:
As you point out, the previous usage of `moment` can be pretty unintuitive. Can you please remind me why this diff right here works? In the previous code, it seems we're adding 1 minute 30 seconds, but now we're just adding 1 minute. Where did the 30 seconds go?

Reply:
Because the original test's behavior was hard to understand and replicate, for this test in particular, I didn't try to replicate the old behavior, but rather I wrote a new history without using `moment.add()`, and in that history it just so happened that the output is 1 min after starting time not 1:30.

I wouldn't think of this diff as changing 1:30 to 1min, but rather rewriting this test, and the new expected value here is 1min.

That being said, when I added those artificial delays you talked about, I added an additional 30 seconds to remove this diff :P